### PR TITLE
Rebuild LLVM using an older GCC

### PR DESCRIPTION
**Pull request by gbaraldi**

### CODE REVIEW

File: L/LLVM/common.jl [hunk 1]
```diff
@@ -307,6 +307,11 @@ if [[ "${target}" == *apple* ]]; then
     fi
 fi
 
+if [[ "${target}" == aarch64-linux* ]]; then
+    CMAKE_C_FLAGS+=(-mno-outline-atomics)
+    CMAKE_CPP_FLAGS+=(-mno-outline-atomics)
+fi
```
Comment (giordano) [inline L/LLVM/common.jl:313]: Should this be gated by version number again?

Comment (gbaraldi) [reply L/LLVM/common.jl:313]: This is gated on the GCC version used to build it

Comment (giordano) [reply L/LLVM/common.jl:313]: But it isn't?

Comment (gbaraldi) [reply L/LLVM/common.jl:313]: Can we do that in the build_tarballs script?

Comment (giordano) [reply L/LLVM/common.jl:313]: https://github.com/JuliaPackaging/Yggdrasil/blob/f59804dc452f9db4c1404c41562d2720e27d81ea/O/OpenBLAS/common.jl#L167

**Review by vchuravy — APPROVED** [review]

**Pull request merged**